mantidproject/mantid
# Add see also for alg documentation

Description of work.

Adds an optional see also section to the algorithm, and makes it visible in the documentation
A first imperfect attempt has been made to update algorithms with seeAlso links.

This is done in the documentation using a RelatedAlgorithms directive.  Note this also replaces the alias directive in order to use less whitespace

**To test:**
Note: this PR contains a LOT of file changes. but this is primarily minor repeated changes to the algs and their rst pages.  The ones that really would benefit from a code review are in the first 4 commits. These refer to a seealso directive, the name of this was changed later to relatedalgorithms to prevent a clash with a built-in seealso directive.


1. Build the docs
1. See the See Also heading for algorithms (most should have some entry)
1. Look at Rebin, this will also have an alias of "rebin" check that looks ok
1. Check both a C++ algorithm (e.g. LoadRaw) and a python alg (e.g. MergeCalFiles) both have see also sections


Fixes #21243

**Release Notes** 
docs/source/release/....framework.rst

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
